src/lib/server/response.ts

```
@@ -1,4 +1,87 @@
+export interface BaseResponse<T> {
+  success: boolean;
+  status: number;
+  message: string;
+  data?: T;
+  errors?: ZodIssue[];
 }
+
+class CustomResponse<T = unknown> extends NextResponse<BaseResponse<T>> {
+  constructor(body?: BodyInit | null, init?: ResponseInit) {
+    super(body, init);
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)